mmv1/third_party/terraform/services/resourcemanager/resource_google_project_service_test.go.erb

```diff
@@ -368,12 +368,14 @@ resource "google_project_service" "test" {
 func testAccProjectService_checkUsage(service string, pid, org string, checkIfServiceHasUsage string) string {
```

Comment:
The new error is because `check_if_service_has_usage_on_destroy` is a parameter in the API (i.e. is url_param_only which implies ignore_read) and wasn't added to ImportStateVerifyIgnore. Non-blocking to resolve in this PR, though.

Unless I'm missing something- it was added to some tests but not others.

Reply:
That's useful context, thanks! I'll copy across to the GH issue and will merge this